src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -85,6 +85,30 @@ public void testComplexSelection()  throws IOException {
         spec.executeTest("testComplexSelection--" + testFile, this);
     }
 
+    @Test
+    public void testUntrimmedVariants() throws IOException {
+        final String testFile = getToolTestDataDir() + "untrimmed.vcf";
```

Comment:
I'm going to be opinionated here and argue against using `IntegrationTestSpec`.  I think the intent of these two tests would be better expressed in code:
```
final ArgumentsBuilder args = new ArgumentsBuilder()
   .addVcf(testFile)
   .addOutput(output)
   .addArgument(StandardArgumentDefinitions.SAMPLE_NAME_SHORT_NAME, "SAMPLE_01")
   .addBoolean("preserve-alleles", false); // this may be overkill since it's the default

runCommandLine(args);

final List<VariantContext> vcs = VariantContextTestUtils.readEntireVCFIntoMemory(output).getRight();

Assert.assertTrue(Comparators.isInOrder(vcs, Comparator.comparingInt(VariantContext::getStart)));
```

The virtues I see in this approach are 1) Only one test file needs to be added/maintained.  2) Doesn't require exact match and therefore less brittle.  3) Intent is explicit.

Reply:
Well argued, I'm convinced!